helpers/internal.psm1

```diff
@@ -370,6 +370,31 @@ Function Get-DisqualifiedNetworksFromMapping {
     Return $Disqualified
 }
 
+Function Get-VDiskStatus {
+    param ( $LogFile )
+
+    $Unhealthy = $false
+    $UnhealthyDisks = @()
+    Write-Host "Getting Virtual Disk Health..."
+    "Getting Virtual Disk Health..." | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+    Get-VirtualDisk | ForEach-Object {
+        if($_.HealthStatus -eq 'Unhealthy'){
+            $Unhealthy = $true
+            $UnhealthyDisks += $_.FriendlyName
+        } 
+    }
+    if($Unhealthy) { 

```

Comment:
Spacing (381 as well). Please pay attention to this so it doesn't need to be corrected later and is easier to read by someone else on the team.